deploy/charts/cert-manager/values.linter.exceptions

```diff
@@ -0,0 +1,6 @@
+value missing from values.yaml: creator
+value missing from values.yaml: nameOverride
+value missing from values.yaml: fullnameOverride
+value missing from templates: crds.enabled
+value missing from templates: crds.keep
+value missing from templates: acmesolver.image.pullPolicy
```

Comment:
The `creator` field is undocumented, since it is used only by our CI scripts.
The fields `nameOverride` and `fullnameOverride` are missing from the values.yaml file and should maybe be documented in the future.
The fields `crds.enabled` and `crds.keep` are used, but those templates are generated dynamically, so this is a false-positive.
The field `acmesolver.image.pullPolicy` is currently not supported, this is something we should fix in the future.